<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sensor1">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sensor2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>450</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sensor3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_agg1">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>350</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_agg2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_initial">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>initial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>sensor資訊</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_sensor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>610</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>車子狀態</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_car">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>540</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>sensor3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>240</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>sensor1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>470</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>sensor2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>370</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>agg1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>agg2</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>16</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>310</y>
     <width>20</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>160</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>sensor4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sensor4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_heat">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>加熱</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_sensor_heat">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sensor1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sensor2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sensor3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sensor4</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_cool">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>降溫</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_sensor_cool">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sensor1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sensor2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sensor3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sensor4</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
